solr/core/src/java/org/apache/solr/cloud/Overseer.java

```diff
@@ -414,7 +414,7 @@ public void run() {
           log.info("Overseer Loop exiting : {}", LeaderElector.getNodeName(myId));
         }
         // do this in a separate thread because any wait is interrupted in this main thread
-        new Thread(this::checkIfIamStillLeader, "OverseerExitThread").start();
+        Thread.ofVirtual.name("OverseerExitThread").start(this::checkIfIamStillLeader);
```

Comment:
Doesn't this need to use `unstarted()` per your table?

Reply:
When the thread was started on the same line, I felt like just using start()

Reply:
Here though I forgot the parentheses after ofVirtual, which is causing an error in the GitHub actions. Let me fix that now

Reply:
Is it OK to just make the commit through GitHub now?

Reply:
Not sure I follow.

Reply:
Nvm, I already did it